script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -412,77 +443,84 @@ But I'm still alive.
 [sha:dcfd639c00f16f3ce1507ff604bfa112a9324abf]{
 -- Page 273, Offset 33980.
 -- なんとか生きてる。
-Somehow, I'm still alive.
+I don't know how, but I'm still alive.
 }
 [sha:ded838f6b32fe18f6ca0b020f3bc5f3b5f5198b7]{
 -- Page 274, Offset 33981.
 -- 体。体は、また先輩から離れている。
-My body is―――once again, separated from Senpai's.
+Where am I...
+ It looks like I've been separated from Senpai again.
 }
 [sha:7cac72cce8cb34b992fb283095e1faeae3300f5a]{
 -- Page 274, Offset 33982.
 -- 肩からは<燻|いぶ>すような火薬の匂い。
-I can smell the scent of fuming gunpowder coming from my shoulder.
+I can smell the acrid scent of gunpowder wafting from my shoulder.
 }
 [sha:86ecfef23071530b3ee4be58fb3b82f464442317]{
 -- Page 274, Offset 33983.
 -- 「あ、ああ、あ」
-"Ah, aah, ah."
+"%{i}Ah%{/i}, %{i}aah%{/i}, %{i}ah%{/i}."
 }
 [sha:4da46d30779b4659405008868d1821a7f463e937]{
 -- Page 274, Offset 33984.
 -- あの銃口が突き刺さった瞬間、引き金を引かれたのか。
-She must have pulled the trigger as soon as she thrust the muzzle into me.
+She must have pulled the trigger the instant she skewered me with that spike.
 }
 [sha:896c7526252a7b6c378922cd7989f152e8560976]{
 -- Page 274, Offset 33985.
 -- それで俺は吹っ飛んで、また先輩と距離が離れている。
-I was blown away, and now we're once again separated by some distance.
+The impact blew me away, and opened up the distance between us again.
```

Comment:
That kinda feels... _too_ explanatory like that IMO? Like if you were talking to a child. I think it would be hard to not read the cause/effect here correctly.

Reply:
I guess, I think it reads fine in context, personally. You can vary it if it's the wording you don't like.

>I see. The impact blew me away, and separated me from her.

>What happened was that the impact blew me away, separating me from her.

etc